Projects/UOContent/Assistants/AssistantHandler.cs

```diff
@@ -84,7 +84,7 @@ public static void AssistVersion(NetState state, CircularBufferReader reader, in
 
         // Instead we are supporting razor community edition.
         var assistVersion = reader.ReadAscii();
-        state.Assistant = assistVersion.Contains(' ') ? assistVersion : $"RazorCE {assistVersion}";
+        state.Assistant = assistVersion.Contains(' ', StringComparison.Ordinal) ? assistVersion : $"RazorCE {assistVersion}";
```

Comment:
We should have a ContainsOrdinal convenience method?

Reply:
Ahh nice